wandb/apis/public/api.py

```diff
@@ -363,6 +364,21 @@
         self._sentry = wandb.analytics.sentry.Sentry(pid=os.getpid())
         self._configure_sentry()
 
+        self._backend: wandb.sdk.backend.backend.Backend | None = None
+        self._service = None
+
+    def _start_backend_service(self):
```

Comment:
This is private, but still please add a docstring that this spin up the wandb-core service process and initializes the resources for handling Api requests there.